kirega/iReporter
# Ch project config setup #162311108

#### What does this PR do?
It includes the config object needed to make it possible to easily switch between objects.
#### Description of Task to be completed?
Add a config class in the instance folders
#### How should this be manually tested?
Clone the repo
`cd iReporter`
`pip install -r requirements.txt`
`python run.py`

You should see the current build enviroment (`development` or `production`) active as set in the `Config object
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162311108
